src/shared/assets/sass/v2/blocks/_inset-text.scss

```diff
@@ -12,3 +12,10 @@
     background: govuk-colour("grey-4");
   }
 }
+
+.inset-text--rename-licence {
+  @extend .govuk-inset-text;
+  border: 2px solid $govuk-text-colour;  
```

Comment:
I wonder if for the border we should use: 
```
border: 1px solid $govuk-border-colour;
```
As this would match more closely some of the other components in the kit?

Reply:
I agree. Will get check with UX design again if this is how they want to proceed.

Reply:
Form has not been redesigned in the last commit